src/open_inwoner/openklant/tests/test_views.py

```diff
@@ -273,11 +357,15 @@ def test_show_detail_for_bsn_with_zaak_reformat_esuite_id(self, m):
             ),
         )
 
-    def test_show_detail_for_kvk_or_rsin(self, m):
+    def test_show_detail_for_kvk_or_rsin(self, m, mock_get_local_kcm_mapping):
         for use_rsin_for_innNnpId_query_parameter in [True, False]:
             with self.subTest(
                 use_rsin_for_innNnpId_query_parameter=use_rsin_for_innNnpId_query_parameter
             ):
+                # Avoid having a `KlantContactMomentLocal` with the same URL for
+                # different users
```

Comment:
I think comments are somewhat exempt from line length rule. Like putting the last two words on the next line is arguably worse then making the line slightly longer.